Adyen.EcommLibrary/Constants/ClientConfig.cs

```diff
@@ -9,8 +9,10 @@ internal class ClientConfig
         public static string CloudApiEndPointTest = "https://terminal-api-test.adyen.com";
         public static string CloudApiEndPointLive = "https://terminal-api-live.adyen.com";
         public static string RecurringApiVersion = "v25";
-        public static string ApiVersion = "v30";
+        public static string ApiVersion = "v40";

```

Comment:
since you are migrating from v30 to v40, I think there must be more changes (difference in v30 and v40) in other models like PaymentRequest etc. and you might require more changes in other models.